helm/ffc-demo-web/jenkins-aws.yaml

```diff
@@ -9,5 +9,3 @@ container:
 ingress:
   endPoint: ffc-demo-web
   server: ffc.aws-int.defra.cloud
-apiGatewayUrl: http://ffc-demo-api-gateway.ffc-demo
```

Comment:
`apiGatewayUrl` should have been nested under `container`. Since `ffc-demo` is the default namespace we use for all services now, I've removed this and updated the main `values.yaml`.